.github/workflows/tools-releases.yml

```diff
@@ -0,0 +1,38 @@
+name: Kubernetes Tools Releases for controller-runtime envtest
+
+on:
+  push:
+    branches:
+      - main
```

Comment:
We might want to switch to a release eventually, this needs to be tested, so for now it might be ok